drivers/at86rf2xx/at86rf2xx.c

```diff
@@ -101,6 +101,17 @@ void at86rf2xx_reset(at86rf2xx_t *dev)
     at86rf2xx_set_page(dev, AT86RF2XX_DEFAULT_PAGE);
 #endif
 
+#if AT86RF2XX_SMART_IDLE_LISTENING
+    tmp = at86rf2xx_reg_read(dev, AT86RF2XX_REG__TRX_RPC);
+    tmp |= (AT86RF2XX_TRX_RPC_MASK__RX_RPC_EN |
+            AT86RF2XX_TRX_RPC_MASK__PDT_RPC_EN |
+            AT86RF2XX_TRX_RPC_MAKK__PLL_RPC_EN |
+            AT86RF2XX_TRX_RPC_MASK__XAH_TX_RPC_EN |
+            AT86RF2XX_TRX_RPC_MASK__IPAN_RPC_EN);
```

Comment:
Would it be reasonable to set these bits inside o `at86rf2xx_set_rxsensitivity()`?

Reply:
Enabling smart idle listening feature requires a proper rx_sensitivity setting.
But rx_sensitivity setting does not require the smart idle listening feature.
So I think this form is better.